src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -137,9 +136,24 @@ const EventEntries = createReactClass({
         );
       }
     });
+  },
+
+  render() {
+    let {group, isShare, project, event, orgId} = this.props;
+
+    let organization = this.getOrganization();
+    let features = organization ? new Set(organization.features) : new Set();
 
     let hasContext =
-      !utils.objectIsEmpty(event.user) || !utils.objectIsEmpty(event.contexts);
+      event && (!utils.objectIsEmpty(event.user) || !utils.objectIsEmpty(event.contexts));
```

Comment:
Do we need these checks everywhere? Aren't we guaranteed to have `event` at this point since it's checked in componentDidMount?

Reply:
`event` is not guaranteed, the check in `cDM` doesn't prevent this from rendering